studio/components/interfaces/Database/Publications/PublicationsTables.tsx

```diff
@@ -29,33 +27,6 @@ const PublicationsTables: FC<Props> = ({
             !meta.excludedSchemas.includes(table.schema) && table.name.includes(filterString)
         )
 
-  const publication = selectedPublication
```

Comment:
can we comment these out instead since we have plans to bring these back
and also leave a comment in the code as to why we're commenting these out